tekton-pipelines/buildpacks-jekyll/tasks/build.yaml

```diff
@@ -94,7 +94,6 @@ spec:
           cd $(resources.inputs.app.path)
           yes | bundle install --quiet
           bundle exec jekyll build
-          cp -rf {project.toml,nginx.conf,buildpack.yml,nginx.d,.nginx.d,nginx.*,httpd.conf,.http.d,httpd.d} $(params.source_subpath) 2>/dev/null
```

Comment:
why this was removed?  If [nginx is not](https://github.com/saritasa-nest/saritasa-frontend-wiki/blob/develop/nginx.conf) copied - and source_subpath points to _site, then you may have issues?

Reply:
So these files should not affect the detection in  a bad way. If nginx is not there - which nginx is used?

Reply:
When we execute `bundle exec jekyll build` bundle copy all content of root folder to `_site`, so it works fine and nginx detect properly. If we also copy files manually we will got error kind of `can't create symbolic link`, i can't specify it couse logs already unavailable.

Reply:
Sorry, but this doesn't look like explanation to me.
Nginx config is not copied from the wiki root, that means the default buildpack nginx config is used?

Reply:
nginx config copied by bundle and we have it inside `_site`